css/css-color/lch-004-ref.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Color 4: Specifying Lab and LCH</title>
+<style>
+    .test { background-color: rgb(75.62% 30.45% 47.56%); width: 12em; height: 12em; } /* lch(50% 50 0) converted to sRGB (happens to be the same as lab(50% 50 0)) */
```

Comment:
it happens because of trigonometry, if either a or b is zero then the chroma is b (or a)

Reply:
Yeah, that was existing text I was just bringing along.